pkg/components/strawberry_controller.go

```diff
@@ -66,13 +66,12 @@ func NewStrawberryController(
 		}
 	}
 
-	l := labeller.Labeller{
-		ObjectMeta:     &resource.ObjectMeta,
-		APIProxy:       ytsaurus.APIProxy(),
-		ComponentLabel: fmt.Sprintf("yt-%s-controller", name),
-		ComponentName:  componentName,
-		Annotations:    resource.Spec.ExtraPodAnnotations,
-	}
+	l := labeller.NewLabellerForGlobalComponent(
+		&resource.ObjectMeta,
+		consts.ComponentType(componentName),
```

Comment:
Shouldn't CHYTType be here?
The point of `ComponentType` is that it is a finite number of well-known values and not expected to created them dynamically.

Reply:
good point, missed it